mfr/extensions/video/templates/viewer.mako

```diff
@@ -1,3 +1,10 @@
+<style>
```

Comment:
* Can you add a `mako` comment `##` explaining why this works? If it is purely magic, then what it does/fixes is fine as well. In this way, others can know why this is a must-have piece without look a the commit message.

* I had worried that `<style>` may not work in `body` but local testing confirmed that MFR put it in the `<header>` tag. 😌 

![video-style](https://user-images.githubusercontent.com/3750414/38044136-e4fa5500-3286-11e8-8d70-72933615a688.png)

Reply:
Its not magic - its just default browser styles add margin and padding on the body tag - I thought this was common knowledge but I can add a comment if we need it. The only reason this is in a style tag and not in a CSS file is because this way we don't need to worry about it affecting other extensions by accident - this will only affect videos. Lastly yep style tags will work anywhere. html5 spec actually doesn't even require head and body tags to exist - the browser automatically inserts them if they're not there.

Reply:
- I see. Thanks for the clarification. I didn't understand why this solve the scroll bar flickering issue and was asking that a comment is helpful here. I tried to use this solution to fix similar scroll bar flickering issue on chrome for images but it didn't work.
- We can not rely on HTML5 since there is IE11. This is where my initial worry came from until I successfully tested this in IE11.